File: lua/1.lua
--project euler 1: find the sum of all multiples of 3 or 5 below 1000
local t = os.clock()

local sum = 0

for i=0,999 do
    if (i%3 == 0) or (i%5 == 0) then
		sum = sum+i
	end
end

print("result: "..sum.." - found in "..(os.clock() - t).." s")

File: cpp/1.cpp
#include <iostream>

using namespace std;

/* Project Euler 1
 * Find the sum of all multiples of 3 and 5 below 1000
 */
int main() {
    int sum = 0;
    for (int i=0; i<1000; i++) {
        if (i%3 == 0 || i%5 == 0) {
            sum += i;
        }
    }
    cout << "Sum: " << sum << "\n";
}
